### ISSUE TITLE
“q_gradient_batch” in ddpg.py

### ISSUE THREAD

**Issue by zcchenvy** [yhyu13/C51-DDPG#1]
line 103: Why change the direction of the gradient？I think this step is not required.  I do not really understand your meaning.

**Comment by yhyu13** [comment]
According to the paper of [ddpg](https://arxiv.org/pdf/1509.02971.pdf) on page 5, the actor needs to be updated by just plain ```q_gradient_batch```. It should be **added** to the actor parameters.

 Bust since the actor train function (see [here](https://github.com/yhyu13/C51-DDPG/blob/c881a026ab8419e215f89ccf2996bbc0b341b4b0/actor_network.py#L67)) from TensorFlow, by default, applying a gradient descent, meaning any gradient given to the optimizer is subtracted. That's why I flipped the gradient so that the end effect is still an adding.

**Comment by zcchenvy** [comment]
> According to the paper of [ddpg](https://arxiv.org/pdf/1509.02971.pdf) on page 5, the actor needs to be updated by just plain `q_gradient_batch`. It should be **added** to the actor parameters.
> 
> Bust since the actor train function (see [here](https://github.com/yhyu13/C51-DDPG/blob/c881a026ab8419e215f89ccf2996bbc0b341b4b0/actor_network.py#L67)) from TensorFlow, by default, applying a gradient descent, meaning any gradient given to the optimizer is subtracted. That's why I flipped the gradient so that the end effect is still an adding.

thank you
